ocpp-v1_6/src/main/java/eu/chargetime/ocpp/model/smartcharging/GetCompositeScheduleConfirmation.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * GetCompositeScheduleConfirmation.java
+ *
+ * Created on 2019-11-21
+ *
+ * Copyright (C) 2019 Volkswagen AG, All rights reserved.
```

Comment:
Just to be on the safe side here. Do you have Volkswagen AGs consent? I would love to tell people that Volkswagen has contributed to my library. :)